recipes/physics-tenpy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "physics-tenpy" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e8239ef7636260cd459c18840b9ba25129f1129344ee1556fdb15faee5f7c0f8 
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<35]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - cython >=0.29
+    - numpy >=1.13
+    - scipy >=0.18
```

Comment:
Is scipy needed in host? Normally packages which are used from python are only needed in run.

Reply:
Yes, it's needed; I quickly tried it and the build fails if scipy is removed.
The reason is that there is a cimport from `scipy.linalg.cython_blas` in the cython file.